### PR TITLE
Fixing Task201014UpdateColumnsValueInIdentifierTableTest

### PR DESCRIPTION
**Pull request by nollymar**
This test was failing in Oracle

### CODE REVIEW

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java [hunk 1]
```diff
@@ -61,17 +62,17 @@ public String getMySQLScript() {
...
-        query.append("update identifier ident,\n")
-                .append("(select distinct temp.identifier myID, temp.mod_user, inode.idate, ")
-                .append("struc.velocity_var_name  from contentlet temp, inode, structure struc,\n")
-                .append("(select identifier, MIN(idate) as idate from contentlet, inode where ")
-                .append("inode.inode=contentlet.inode group by identifier) custom_select\n")
-                .append("where temp.identifier=custom_select.identifier\n")
-                .append("and inode.inode=temp.inode and inode.idate=custom_select.idate \n")
-                .append("and temp.structure_inode = struc.inode) my_query\n")
-                .append("set ident.owner=my_query.mod_user, ident.create_date=my_query.idate,\n")
+        query.append("UPDATE identifier ident,\n")
```
Comment (nollymar) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java:65]: changing format only

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java [hunk 2]
```diff
@@ -31,14 +31,15 @@ public String getPostgresScript() {
...
         //update contentlets
-        query.append("update identifier set owner=mod_user, create_date=idate, asset_subtype=velocity_var_name from\n")
-                .append("(select distinct temp.identifier myID, temp.mod_user, inode.idate, struc.velocity_var_name  from contentlet temp, inode, structure struc,\n")
-                .append("(select identifier, MIN(idate) idate from contentlet, inode \n")
-                .append("where inode.inode=contentlet.inode group by identifier) custom_select  \n")
-                .append("where temp.identifier=custom_select.identifier\n")
-                .append("and inode.inode=temp.inode and inode.idate=custom_select.idate \n")
-                .append("and temp.structure_inode = struc.inode) my_query\n")
-                .append(" where  id=myID;\n");
+        query.append("UPDATE identifier SET owner=mod_user, create_date=idate, asset_subtype=velocity_var_name from\n")
```
Comment (nollymar) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java:34]: changing format only

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java [hunk 3]
```diff
@@ -127,45 +138,56 @@ public String getH2Script() {
...
+                .append("WHERE  myID = tt.id);").toString();
+
+
     }
 
     private String getQueryToUpdateNonContentletsPostgres(final String tableName){
         final StringBuilder query = new StringBuilder();
-        return query.append("update identifier set owner=iowner, create_date=idate from\n")
-                .append("(select distinct temp.identifier myID, owner iowner, inode.idate idate from ")
+        return query.append("UPDATE identifier SET owner=iowner, create_date=idate FROM\n")
```
Comment (nollymar) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java:169]: changing format only

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java [hunk 4]
```diff
@@ -127,45 +138,56 @@ public String getH2Script() {
...
+                .append(" WHERE temp.identifier=custom_select.identifier\n")
+                .append(" AND inode.inode=temp.inode AND inode.idate=custom_select.idate) my_query\n")
+                .append(" WHERE id=myID;\n").toString();
     }
 
     private String getQueryToUpdateNonContentletsMySQL(final String tableName){
         final StringBuilder query = new StringBuilder();
-        return query.append("update identifier ident,\n")
-                .append("(select distinct temp.identifier myID, owner iowner, inode.idate idate  from ")
+        return query.append("UPDATE identifier ident,\n")
```
Comment (nollymar) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java:182]: changing format only

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTableTest.java [hunk 5]
```diff
@@ -96,10 +96,13 @@ private boolean areValuesEqual(final Identifier expectedResult, final Map<String
                 ((Date)result.get("create_date")).getTime()).toLocalDateTime().truncatedTo(
                 ChronoUnit.SECONDS);
 
-        return result.get("owner").equals(expectedResult.getOwner()) &&
+        return ((expectedResult.getOwner() == null && result.get("owner") == null) || result
```
Comment (dsilvam) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTableTest.java:99]: This looks very confusing. I'd try to refactor maybe creating small methods with good names.

Comment (nollymar) [reply dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTableTest.java]: Done

**Review by dsilvam — APPROVED** [review]

**Review by erickgonzalez — APPROVED** [review]

**Pull request merged**